app/react/actions/index.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+export const LOAD_MESSAGES = 'LOAD_MESSAGES';
+export const SET_ACTIVE_ROOM = 'SET_ACTIVE_ROOM';
+export const SUBMIT_MESSAGE = 'SUBMIT_MESSAGE';
+export const DISPLAY_NEW_MESSAGE = 'DISPLAY_NEW_MESSAGE';
+
+const BASE_URL = 'http://localhost:3000';
+
+export function loadMessages(roomId) {
+  const url = `${BASE_URL}/messages?room_id=${roomId}`;
```

Comment:
also, instead of appending the query params to url, you can use axios' `params:` key. This way it will take care of things like proper escaping etc...

For example:
```javascript
const api = axios.create({
  baseURL: BASE_URL,
});

export function loadMessages(roomId) {
  const request = axios.get("/messages", { params: { room_id: roomId } });
  // rest of the function
```